src/components/EpisodesNav/renderUtils.jsx

```diff
@@ -37,10 +56,39 @@ function renderPaginationLink(data, index){
 
 	const episodeId = UTILS.convertToSafeString(title, "-");
 
-	const isActive = index === activeIndex;
+	const isActive 	= index === activeIndex;
+
+	let opacity;
+	if(isActive) opacity = 1;
+	else {
+		const decayDistance = 4;
+		const decayRate     = 1/decayDistance;
+		const fadeOffset    = Math.abs(index - activeIndex);
+		
+		opacity = 1 - (fadeOffset * decayRate);
+	}
+	
+	const offset = index - activeIndex;
+	
+	let axis, factor;
+	if(isLarge){
+		axis   = "Y";
+		factor = 1.5;
+	}
+	else{ 
+		axis   = "X";
+		factor = 1;
+	}
+
+	const style = {
+		transform: `translate${axis}(${factor*offset}em)`,
```

Comment:
maybe even 
```const translation = `${factor * offset}em`;``` 
so that the transform can just be `${translate}(${translation})`? I dunno. whatever you dig.